src/Microsoft.AspNet.Server.Kestrel/App.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
+  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
+  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>
```

Comment:
This file can't really be used in general with the KRE. It won't work because it won't be found